modules/ROOT/pages/_partials/commons/common-releasenotes.adoc

```diff
@@ -22,12 +22,10 @@ ifdef::param-2-8-1[]
 [#maint-latest]
 [#maint-2.8.1]
 == Release 2.8.1 (November 2020)
-This maintenance release addresses a backward-compatibility issue (https://issues.couchbase.com/browse/CBL-1406[CBL-1406]), which affects:
+Couchbase Lite on {param-title} 2.8.1, supersedes version 2.8.0, which was released in October 2020.
+It addresses a backward-compatibility issue (https://issues.couchbase.com/browse/CBL-1406[CBL-1406]).
 
-* Couchbase Lite on Java
-* Couchbase Lite on Android (Java)
-
-If you develop apps using those products, and those apps could potentially access databases created by a mix of Release 2.7 and Release 2.8 versions, then you should apply Release 2.8.1 as soon as practicable.
+*If you have already upgraded to 2.8.0 we strongly recommend that you upgrade to version 2.8.1 at the earliest opportunity.*
```

Comment:
Prefer suggestion from Amarantha
"Couchbase Lite for Android 2.8.1, released in November 2020, supersedes version 2.8.0 released earlier this year. If you are currently using version 2.8.0, we strongly recommend that you upgrade to version 2.8.1 at the earliest."